docs/src/pages/components/avatars/GroupAvatars.tsx

```diff
@@ -1,15 +1,47 @@
 import * as React from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
 import Avatar from '@material-ui/core/Avatar';
 import AvatarGroup from '@material-ui/lab/AvatarGroup';
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      '& > *': {
+        margin: theme.spacing(1),
+      },
+    },
+  }),
+);
+
 export default function GroupAvatars() {
+  const classes = useStyles();
+
   return (
-    <AvatarGroup max={4}>
-      <Avatar alt="Remy Sharp" src="/static/images/avatar/1.jpg" />
-      <Avatar alt="Travis Howard" src="/static/images/avatar/2.jpg" />
-      <Avatar alt="Cindy Baker" src="/static/images/avatar/3.jpg" />
-      <Avatar alt="Agnes Walker" src="/static/images/avatar/4.jpg" />
-      <Avatar alt="Trevor Henderson" src="/static/images/avatar/5.jpg" />
-    </AvatarGroup>
+    <div className={classes.root}>
+      <AvatarGroup max={4}>
+        <Avatar alt="Remy Sharp" src="/static/images/avatar/1.jpg" />
+        <Avatar alt="Travis Howard" src="/static/images/avatar/2.jpg" />
+        <Avatar alt="Cindy Baker" src="/static/images/avatar/3.jpg" />
+        <Avatar alt="Agnes Walker" src="/static/images/avatar/4.jpg" />
+        <Avatar alt="Trevor Henderson" src="/static/images/avatar/5.jpg" />
+      </AvatarGroup>
+      <AvatarGroup max={4} variant="rounded">
```

Comment:
It breaks the inline preview. I think that it makes the demo more complex that it needs to be. I think that we can rely on the new class name tests. I also think that for the documentation the API section is enough.

Reply:
I would agree, nothing new is being shown here for the grouping behaviour.

Reply:
oh ok! i'll revert it back 👌🏼